### PR TITLE
Added please give feedback "log".

### PR DESCRIPTION
**Pull request by wgyt**
Signed-off-by: William (Wgyt) Horning <williamhorning@wgyt.tk>

Resolves https://canary.discord.com/channels/751206349614088204/751207339545198682/868605962993418312

### Changes

- New variable called plsGiveFeedback that has a l10n'd link to the feedback page
- We append that to extraContainer
- This makes a notice

### Reason for changes

@WorldLanguages told me to

### Tests

Works on chrome:

Before clear:

image

After clear:

image

### CODE REVIEW

File: addons/debugger/userscript.js [hunk 1]
```diff
@@ -217,13 +217,19 @@ export default async function ({ addon, global, console, msg }) {
     innerText: msg("close"),
   });
 
+  const plsGiveFeedback = Object.assign(document.createElement("a"), {
+    innerText: msg("pls-give-feedback"),
```
Comment (FunctionalMetatable) [inline addons/debugger/userscript.js:221]: Did you add the locales?

Comment (wgyt) [reply addons/debugger/userscript.js:221]: How would I do that?

Comment (wgyt) [reply addons/debugger/userscript.js:221]: Done

**Comment by GrahamSH-LLK** [comment]
I think I'm gonna re-implement

**Comment by wgyt** [comment]
With latest changes:
image
image

**Pull request closed without merging**